### ISSUE TITLE
Talkback or TTS crashes under netflix in 4.1.1

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#160]
```
What steps will reproduce the problem?
1. When talkback is running open netflix on your device, attempt to navagate 
around the app using swipe or explore by touch.
2. When navagating around netflix either talkback or TTS crashes and there is 
no speach, this is not happening in other apps I have that are simular like the 
CW TV app.
3. press power to have the device go to sleep and wake it up then unlock the 
screen.  This will bring back speach adventually.

What is the expected output? What do you see instead?
I am expecting to see netflix's list of movies and TV shows on my device.

What version of the product are you using? On what operating system?
I am using the latest talkback just released to the goolge play store as of 
last night and my device is the samsung galaxy s3 running android jelly bean 
version 4.1.1 from US t-mobile "T,9,9,9"

Please provide any additional information below.


```

Original issue reported on code.google.com by `danny.ma...@gmail.com` on 30 Nov 2012 at 3:10

**Comment by GoogleCodeExporter** [comment]
```
There's no crash happening here.  It appears that the Netflix application 
itself is muting and locking the media audio stream that text to speech uses.  
The lock on the stream is not released until the Netflix application is 
terminated.

TalkBack does not have the ability to override this behavior, so this should be 
reported to Netflix.
```

Original comment by `caseybur...@google.com` on 30 Nov 2012 at 10:08
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
Issue 175 has been merged into this issue.
```

Original comment by `caseybur...@google.com` on 12 Feb 2013 at 3:23

**Comment by GoogleCodeExporter** [comment]
```
Issue 175 has been merged into this issue.
```

Original comment by `caseybur...@google.com` on 20 Feb 2013 at 12:57
